ehrapy/api/data/__init__.py

```diff
@@ -2,7 +2,7 @@
 from ehrapy.api.data.datasets import Datasets
 
 
-def mimic_2(encode: bool = False):
+def mimic_2_demo(encode: bool = False):
```

Comment:
That's not quite correct.

We are using the full mimic_2_IAC dataset. We should actually name this mimic_2_iac or MIMIC_II_IAC or something along those lines.

The MIMIC-III dataset that we are using is explicitly only a demo dataset. It is not the full one.